src/particles/plasma/MultiPlasma.H

```diff
@@ -53,6 +53,16 @@ public:
         bool deposit_jz, bool deposit_rho, bool deposit_chi, bool deposit_rhomjz,
         amrex::Vector<amrex::Geometry> const& gm, int const lev);
 
+    /** Loop over plasma species and depose their currents into the current 2D slice in fields
```

Comment:
Just for consistency
```suggestion
    /** Loop over plasma species and deposit their temperature into the current 2D slice in fields
```